integration-tests/deployment/ccip/test_helpers.go

```diff
@@ -55,12 +59,30 @@ func Context(tb testing.TB) context.Context {
 	return ctx
 }
 
-type DeployedTestEnvironment struct {
+type DeployedOnChainEnv struct {
 	Ab           deployment.AddressBook
 	Env          deployment.Environment
```

Comment:
hm Env.OffchainClient would not be onchain only?

Reply:
Does `DeployedEnv` sound alright?

Reply:
Or can change this to `DeployedTestEnvironment` and update the following with `DeployedTestEnvironmentWithMemoryNodes`
```
type DeployedTestEnvironment struct {
	DeployedOnChainEnv
	Nodes map[string]memory.Node
}
```